path_config.go

```diff
@@ -286,6 +322,8 @@ type jwtConfig struct {
 	OIDCDiscoveryCAPEM   string   `json:"oidc_discovery_ca_pem"`
 	OIDCClientID         string   `json:"oidc_client_id"`
 	OIDCClientSecret     string   `json:"oidc_client_secret"`
+	OIDCResponseMode     string   `json:"oidc_response_mode"`
+	OIDCResponseTypes    []string `json:"oidc_response_types`
```

Comment:
```suggestion
	OIDCResponseTypes    []string `json:"oidc_response_types"`
```

Reply:
This is addressed in https://github.com/hashicorp/vault-plugin-auth-jwt/pull/99 if we merge it.